superset/assets/javascripts/SqlLab/components/ResultSet.jsx

```diff
@@ -211,12 +211,13 @@ class ResultSet extends React.PureComponent {
     }
     if (query.cached) {
       return (
-        <a
-          href="#"
+        <Button
```

Comment:
I think there was a bug in this functionality that query wasn't using the cta schema, could u please fix it in this PR ?